spec/jobs/upload_documents_job_spec.rb

```diff
@@ -13,17 +13,17 @@
     let!(:service) { double }
 
     it "calls UploadDocumentsService" do
-      allow(service).to receive(:call).and_return(service)
+      expect(service).to receive(:call)
 
       allow(UploadDocumentsService).to receive(:new)
```

Comment:
I'd recommend changing this to an `expect` as well

Reply:
@allbecauseyoutoldmeso also had expect there before Rubocop's suggestion. 

When I make the change Rubocop tells me:
[RSpec/StubbedMock: Prefer allow over expect when configuring a response.
](https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/StubbedMock)

Do you still prefer the change?

Reply:
`service` only receives `#call` if `UploadDocumentsService` receives `.new`, so I guess the first expectation covers both 🤔

Reply:
I think what the rule is saying is that you do a `allow` for the `and_return` and then an `expect` to set an expectation. This seems arbitrarily verbose and I'd recommended disabling `RSpec/StubbedMock` in the rubocop config.